man/os-release.xml

```diff
@@ -392,6 +392,35 @@
           for light blue, or <literal>ANSI_COLOR="0;38;2;60;110;180"</literal> for Fedora blue.
           </para></listitem>
         </varlistentry>
+
+        <varlistentry>
+          <term><varname>VENDOR_NAME=</varname></term>
+
+          <listitem><para>The name of the OS vendor. This is the name of the organization or company which
+          produces the OS. This field is optional.</para>
```

Comment:
maybe add that this is supposed to be a string intended for humans to read (as opposed to an identifier that programs should consume).

after all we have VERSION and VERSION_ID where the former is human reable version and the latter is the computer processable version. Given we make the distinction there, we should hence say for VENDOR_NAME= what it is.